pkg/plugins/resources/postgres/pgx_store.go

```diff
@@ -26,6 +26,8 @@ type pgxResourceStore struct {
 
 var _ store.ResourceStore = &pgxResourceStore{}
 
+var CustomizePgxConfig = func(pgxConfig *pgxpool.Config) {}
```

Comment:
Same anyway to make this more pluggable at a higher level? (It's really hard with these globals to piece together what's required where)